public/modules/custom/grants_metadata/src/TypedData/Definition/TyollisyysavustusHakemusDefinition.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace Drupal\grants_metadata\TypedData\Definition;
+
+use Drupal\Core\TypedData\ComplexDataDefinitionBase;
+use Drupal\Core\TypedData\DataDefinition;
+
+/**
+ * Define Tyollisyysavustushakemus data.
+ */
+class TyollisyysavustusHakemusDefinition extends ComplexDataDefinitionBase {
+
+  use ApplicationDefinitionTrait;
+
+  /**
+   * Base data definitions for all.
+   *
+   * @return array
+   *   Property definitions.
+   */
+  public function getPropertyDefinitions(): array {
+    if (!isset($this->propertyDefinitions)) {
+
+      $info = &$this->propertyDefinitions;
+
+      foreach ($this->getBaseProperties() as $key => $property) {
+        $info[$key] = $property;
+      }
+
+      $info['members_applicant_person_local'] = DataDefinition::create('string')
+        ->setLabel('activitiesInfoArray=>membersApplicantPersonLocal')
```

Comment:
Tän labelin vois kopsia webformilta suoraan. Toi mitä tos nyt on, on jämiä mun alkuperäsistä testailuista ku testailin miten tota vois käyttää. Ei näin ainakaa :)